tests/memberclient.py

```diff
@@ -48,8 +48,10 @@ def run(args):
         return Calls:call("raw_puts", p)
         """
 
-        LOG.info("Proposal to add a new member")
-        infra.proc.ccall("./keygenerator", "--name=member3")
+        LOG.info("Proposal to add a new member (with different curve)")
+        infra.proc.ccall(
+            "./keygenerator.sh", "member3", infra.ccf.ParticipantsCurve.secp256k1.name
```

Comment:
This assumes we're still on the default curve, so that secp256k1 is a different curve. Can we do something like `infra.ccf.ParticipantsCurve( (args.default_curve.value + 1) % ...).name`?

Reply:
Quick hunt in Python for a neatish way of saying this:

```
class Foo(enum.IntEnum):
   bar = 0
   baz = enum.auto()
   bat = enum.auto()

def next(e):
  return Foo( (e.value+1) % len(Foo) )

>>> next(Foo.bar)
<Foo.baz: 1>
>>> next(Foo.baz)
<Foo.bat: 2>
>>> next(Foo.bat)
<Foo.bar: 0>
```